pkg/controller/backend.go

```diff
@@ -981,15 +981,19 @@ func createTransportServiceDecl(cfg *ResourceConfig, sharedApp as3Application) {
 	if cfg.Virtual.Mode == "standard" {
 		if cfg.Virtual.IpProtocol == "udp" {
 			svc.Class = "Service_UDP"
-		} else {
+		} else if cfg.Virtual.IpProtocol == "tcp" {
```

Comment:
This would be a breaking change for existing users. Should we have tcp as default IP protocol ?   @trinaths